app/src/org/commcare/tasks/ZipTask.java

```diff
@@ -200,21 +72,24 @@ private boolean zipTargetFolder(File targetFilePath, String zipFile) throws IOEx
         return false;
     }
 
-    private boolean zipFolder(File[] files, String zipFile, ZipOutputStream zos) throws IOException {
-        Log.d(TAG, "827 zipping folder with files: " + files[0] + ", zipFile: " + zipFile);
+    private void zipInstanceFolder(File[] toBeZippedFiles, String zipFilePath, ZipOutputStream zos)
+            throws IOException {
+        Log.d(TAG, "Zipping instance folder with files: " + Arrays.toString(toBeZippedFiles)
+                + ", zipFilePath: " + zipFilePath);
+
         int BUFFER_SIZE = 1024;

```

Comment:
Is there a reason you aren't using the default buffer size specified by `BufferedInputStream`?